src/status_im/stickers/core.cljs

```diff
@@ -86,11 +86,16 @@
                  (re-frame/dispatch [:stickers/pack-owned pack-id]))}))})))})))
 
 (fx/defn init-stickers-packs
-  [{:keys [db]}]
-  (let [sticker-packs (get-in db [:multiaccount :stickers/packs-installed])]
-    {:db (assoc db
-                :stickers/packs-installed sticker-packs
-                :stickers/packs sticker-packs)}))
+  [{:keys [db] :as cofx}]
+  (let [sticker-packs (get-in db [:multiaccount :stickers/packs-installed])
+        pending-packs (get-in db [:multiaccount :stickers/packs-pending] #{})]
+    (fx/merge cofx
+              {:db (assoc db
+                          :stickers/packs-installed sticker-packs
+                          :stickers/packs sticker-packs
+                          :stickers/packs-pending pending-packs)}
+              #(when (not-empty pending-packs)
+                 {:stickers/set-pending-timeout-fx nil}))))
```

Comment:
It could be just
```
(cond-> {:db (assoc db
                    :stickers/packs-installed sticker-packs
                    :stickers/packs sticker-packs
                    :stickers/packs-pending pending-packs)}
  (not-empty pending-packs)
  (assoc :stickers/set-pending-timeout-fx nil))
```